Lib/dis.py

```diff
@@ -107,13 +112,19 @@ def pretty_flags(flags):
     return ", ".join(names)
 
 def _get_code_object(x):
-    """Helper to handle methods, functions, generators, strings and raw code objects"""
+    """Helper to handle methods, functions, generators,
```

Comment:
Hm. What about """Helper to handle methods, raw or attributive code objects, and strings""" (?) That wouldn't need to be changed if new objects with `.??_code` attributes are invented.